src/main/java/Equation.java

```diff
@@ -0,0 +1,62 @@
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(
+        name = "Equation",
+        urlPatterns = {"/calc/equation"}
+)
+public class Equation extends HttpServlet {
+void func(PrintWriter out, HttpServletResponse resp) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4535407)